tests/testSpanSets.cc

```diff
@@ -477,6 +477,25 @@ std::pair<std::shared_ptr<afwGeom::SpanSet>, std::shared_ptr<afwGeom::SpanSet>>
                      std::shared_ptr<afwGeom::SpanSet>>(firstSpanSet, secondSpanSet);
 }
 
+std::pair<lsst::afw::image::Mask<lsst::afw::image::MaskPixel>, std::shared_ptr<afwGeom::SpanSet> >
+ makeMaskAndSpanSetForOperationTests() {
+    // Create three overlapping regions in a mask, and a SpanSet to use in set operation tests
+    // This box will range from 0 to 4 in y and 0 to 4 in x
```

Comment:
the first argument in From Shape is a radius. So spanSetFromShape(2, BOX) will make a box that ranges from -2 to 2 in both x and y. Shifting by 2,2 will bring the lower corner up to 0,0 and the upper corner to 4,4.